install

```diff
@@ -85,6 +85,14 @@ pause
 
 # End dependencies via Homebrew
 
+# Begin Nerd Font
+
+if [ $TERM_PROGRAM == "Apple_Terminal" ]; then
```

Comment:
I don't think we need this to be conditional. Even if I'm not using Terminal.app, I think I'd want Terminal.app configured for me to work properly in case I did need it.

Reply:
So without this, if you try to update Terminal's settings from iTerm2 (for the first time) you'll get a prompt similar to the one below - feels a bit more invasive, but I guess it's okay if we explain what's happening?

![image](https://user-images.githubusercontent.com/40244233/188850686-d7a5fc73-08a7-481f-b4a4-9d1e0af3cfa7.png)

Reply:
I think take the same approach we do with warning that Homebrew might ask for passwords? Feels ok to me.